include/dsAudio.h

```diff
@@ -211,52 +211,6 @@ dsError_t  dsAudioPortTerm();
  */
 dsError_t  dsGetAudioPort(dsAudioPortType_t type, int index, intptr_t *handle);
 
```

Comment:
dsAudioOutIsConnected - This is used only for the HEADPHONE port, not for the other ports. Also used only for the sink devices. Update the function description

Reply:
For now it is used for Headphones. 
Changing the generic API specific to headphone needs confirmation from Module owners